cvat-canvas/src/typescript/canvasView.ts

```diff
@@ -120,14 +121,15 @@ export class CanvasViewImpl implements CanvasView, Listener {
         }
     }
 
-    private onEditDone(state: any, points: number[]): void {
-        if (state && points) {
+    private onEditDone(state: any, points: number[], annotation_type: string): void {
```

Comment:
I suppose if a shape was edited it always become manual. Right? It's why I believe we do not need this argument and return value in event details below in this case.